packages/framework/esm-styleguide/src/icons/icons.tsx

```diff
@@ -698,6 +699,48 @@ export const RadiologyIcon = ImageMedicalIcon;
  */
 export const ShoppingCartAddItemIcon = ShoppingCartArrowDownIcon;
 
+/**
+ * This is a utility component that takes an `icon` and render it if the sprite for the icon
+ * is available. The goal is to make it easier to conditionally render configuration-specified icons.
+ *
+ * @example
+ * ```tsx
+ *   <MaybeIcon icon='omrs-icon-baby' className={styles.myIconStyles} />
+ * ```
+ */
+export const MaybeIcon = memo(
```

Comment:
```suggestion
export const MaybeIcon = memo(
```

nit: Maybe `LoadableIcon` reads better here?